src/main/java/mongodb/Main.java

```diff
@@ -0,0 +1,220 @@
+package mongodb;
+
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.MongoException;
+import com.mongodb.client.*;
```

Comment:
You might consider either:
1) Using wildcards in all your imports for brevity
2) Using specific package names instead of wildcards for clarity

Reply:
Going to change this for clarity -- unfortunately, IntelliJ's default settings are rather aggressive about wildcarding.